src/modules/mc_pos_control/mc_pos_control_params.c

```diff
@@ -454,6 +459,8 @@ PARAM_DEFINE_FLOAT(MPC_ACC_HOR, 5.0f);
 /**
  * Slow horizontal manual deceleration for manual mode
  *
+ * Note: This is only used when MPC_POS_MODE is set to 1 or 2.
```

Comment:
No smoothing in sport mode. See other comment.
```suggestion
 * Note: This is only used when MPC_POS_MODE is set to 1.
```